mingw-w64-mysql_client/PKGBUILD

```
@@ -0,0 +1,40 @@
+# Maintainer: taozuhong <taozuhong@gmail.com>
+
+_realname=mysql
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}-client"
+pkgver=8.0.25
+pkgrel=1
+pkgdesc="MySQL client libraries (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64')
+url="https://dev.mysql.com/doc/refman/8.0/en/"
+license=("GPL-2.0-only with Universal-FOSS-exception-1.0")
```

```suggestion
license=("GPL-2.0-only" "Universal-FOSS-exception-1.0")
```

I think these two should be separated since they are separate licenses